masterlist.yaml

```
@@ -25524,19 +25524,36 @@ plugins:
     msg:
       - <<: *patchIncluded
         subs: [ 'Creation Club - Fishing' ]
-        condition: 'active("ccBGSSSE001-Fish.esm") and not active("SurWR - Fishing.esp") and checksum("SurWR.esp", 538498A4)'
+        condition: 'active("ccBGSSSE001-Fish.esm") and not active("SurWR - Fishing.esp") and checksum("SurWR.esp", C1861BE4)'
```

I can correct these in a small correction commit, so they should be removed from this PR to keep it focused:

```suggestion
        condition: 'active("ccBGSSSE001-Fish.esm") and not active("SurWR - Fishing.esp") and checksum("SurWR.esp", 538498A4)'
```